sdk/instrumentation/net/http/handler.go

```diff
@@ -65,7 +66,11 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	url := r.URL.String()
-	span.SetAttribute("http.url", url)
+	if strings.Contains(url, "://") {
```

Comment:
fixing this issue in the instrumentation itself: https://github.com/Traceableai/goagent-src/blob/2320366e224b742d023236680e7b460e4a17630f/filter/traceable/filter.go#L352